pkgs/applications/office/espanso/default.nix

```diff
@@ -39,7 +39,7 @@ assert stdenv.isDarwin -> !x11Support;
 assert stdenv.isDarwin -> !waylandSupport;
 rustPlatform.buildRustPackage rec {
   pname = "espanso";
-  version = "2.2.2-pre";
+  version = "2.2-unstable-2024-05-14";
```

Comment:
I think this line should go into the previous commit, the rest of the changes into the next one.